src/javascript/app/pages/cashier/cashier.js

```diff
@@ -12,7 +12,9 @@ const Cashier = (() => {
 
     const showContent = () => {
         Client.activateByClientType();
-        Header.upgradeMessageVisibility(); // To handle the upgrade buttons visibility
+        if (Client.getUpgradeInfo().type !== 'financial') {
```

Comment:
I think we should act similarly to cashier page and remove all upgrade buttons from this page.

Reply:
I mean, from payment methods page.